dev/sparktestsupport/modules.py

```diff
@@ -747,6 +747,7 @@ def __hash__(self):
         "pyspark.sql.connect.readwriter",
```

Comment:
I think we should add `avro` into `dependencies` above.

Reply:
thanks, let me have a try

I thought `pyspark_sql` already depends on avro, so didn't touch it

Reply:
oh sorry. Yes, you're correct.